### PR TITLE
#8472 - Checkout package changes for rn

### PR DESCRIPTION
**Pull request by ft-lucky**

### CODE REVIEW

File: src/components/paymentContainer/index.tsx [hunk 1]
```diff
@@ -90,6 +91,7 @@ export const PaymentContainer = ({
   onGetPaymentDataError = () => {},
   onPaymentError = () => {},
   stripeCardOptions = {},
+  disableZipSection,
```
Comment (ft-worker) [inline src/components/paymentContainer/index.tsx:94]: Add default value.

File: src/components/billing-info-container/index.tsx [hunk 2]
```diff
@@ -515,17 +518,19 @@ export const BillingInfoContainer = ({
             <div className={`billing-info-container ${theme}`}>
               {!isLoggedIn && (
                 <div className="account-actions-block">
-                  <div>
-                    Got a <strong>TICKET</strong>FAIRY account?
-                  </div>
+                  <div>{accountInfoTitle}</div>
```
Comment (ft-worker) [inline src/components/billing-info-container/index.tsx:521]: Make corresponding changes for HOX.

File: src/components/billing-info-container/index.tsx [hunk 3]
```diff
@@ -94,6 +94,7 @@ export interface IBillingInfoPage {
   buttonName?: string;
   theme?: 'light' | 'dark';
   isLoggedIn?: boolean;
+  accountInfoTitle: any;
```
Comment (ft-worker) [inline src/components/billing-info-container/index.tsx:97]: Add type(s).

File: src/api/index.ts [hunk 4]
```diff
@@ -26,7 +26,7 @@ if (
   isWindowDefined &&
   !discardedOrigins.some(el => origin && origin.includes(el))
 ) {
-  ttfHeaders['X-Source-Origin'] = 'houseofx.nyc'
+  ttfHeaders['X-Source-Origin'] = 'restlessnit.es'
```
Comment (ft-worker) [inline src/api/index.ts:29]: Set `X-Source-Origin` from props in `setConfigs` function call.

File: src/api/index.ts [hunk 5]
```diff
@@ -5,7 +5,7 @@ import { CONFIGS } from '../utils'
 
 const isWindowDefined = typeof window !== 'undefined'
 const discardedOrigins = [
-  'http://localhost',
+  // 'http://localhost',
```
Comment (ft-worker) [inline src/api/index.ts:8]: Remove `discardedOrigins` and connected functionality.

**Review by ft-worker — CHANGES_REQUESTED** [review]

File: src/components/billing-info-container/index.tsx (hunk 2 above)
Comment (ft-lucky) [reply src/components/billing-info-container/index.tsx:521]: PR for HOX - https://github.com/HouseOfXNYC/houseofxsite/pull/119

File: src/components/confirmationContainer/index.tsx [hunk 6]
```diff
@@ -39,15 +41,16 @@ export const ConfirmationContainer = ({
 
   useEffect(() => {
     (async () => {
-      const dataEncoded =
-        typeof window !== 'undefined' &&
-        window.localStorage.getItem('checkoutData')
-      if (dataEncoded) {
+      const dataEncoded = (typeof window !== 'undefined' && window.localStorage.getItem('checkoutData')) || ''
+      if (dataEncoded || orderHash) {
         const dataDecoded = JSON.parse(dataEncoded)
```
Comment (ft-worker) [inline src/components/confirmationContainer/index.tsx:46]: `const dataDecoded = dataEncoded ? JSON.parse(dataEncoded) : { hash: orderHash }`

**Review by ft-worker — CHANGES_REQUESTED** [review]

**Review by ft-worker — APPROVED** [review]

**Pull request merged**